Builds/CMake/RippledCov.cmake

```diff
@@ -2,26 +2,10 @@
    coverage report target
 #]===================================================================]
 
-ProcessorCount(PROCESSOR_COUNT)
-
 if (coverage)
-  if (DEFINED CODE_COVERAGE_TEST_PARALLELISM)
-    set(TEST_PARALLELISM ${CODE_COVERAGE_TEST_PARALLELISM})
-  else()
-    set(TEST_PARALLELISM ${PROCESSOR_COUNT})
-  endif()
-
-  if (DEFINED CODE_COVERAGE_REPORT_FORMAT)
-    set(CODE_COVERAGE_FORMAT ${CODE_COVERAGE_REPORT_FORMAT})
-  else()
-    set(CODE_COVERAGE_FORMAT html-details)
-  endif()
-
-  if (DEFINED CODE_COVERAGE_EXTRA_ARGS)
-    set(GCOVR_ADDITIONAL_ARGS ${CODE_COVERAGE_EXTRA_ARGS})
+  set(GCOVR_ADDITIONAL_ARGS ${coverage_extra_args})
```

Comment:
Can we add a comment here like the following, assuming it is correct?

```
# The instructions for these commands come from the `CodeCoverage` module,
# which was copied from https://github.com/bilke/cmake-modules,
# commit <sha1>.
```

I think this file should hold the command for `include(CodeCoverage)` too.

Reply:
there's a significant number of local changes, I will point this out.